test/ExportQueryExecutionTreeTest.cpp

```diff
@@ -26,6 +26,7 @@ std::string runQueryStreamableResult(const std::string& kg,
   QueryPlanner qp{qec};
   auto pq = SparqlParser::parseQuery(query);
   auto qet = qp.createExecutionTree(pq);
+  qet.isRoot() = true;
```

Comment:
I think we should if possible get back to the `doesn't matter if we don't care` semantics.